Formula/tensorflow_c.rb

```diff
@@ -0,0 +1,42 @@
+class TensorflowC < Formula
+  desc "C interface for Google's OS library for Machine Intelligence"
+  homepage "https://www.tensorflow.org/"
+  url "https://github.com/tensorflow/tensorflow", :using => :git,
+    :tag => "v1.0.0", :revision => "07bb8ea2379bd459832b23951fb20ec47f3fdbd4"
+
+  depends_on "bazel" => :build
+  depends_on "pkg-config" => :run
+
+  def install
+    system 'echo "\n\n\n\n\n\n\n\n\n" | ./configure'
+    system "bazel", "build", "--compilation_mode=opt", "--copt=-march=native", "tensorflow:libtensorflow_c.so"
+    lib.install "bazel-bin/tensorflow/libtensorflow_c.so"
+    cp "tensorflow/c/c_api.h", "tensorflow_c_api.h"
+    include.install "tensorflow_c_api.h"
+    pc = <<-EOF.gsub(/^\s+/, "")
+      Name: tensorflow_c
+      Description: Tensorflow c lib
+      Version: #{version}
+      Libs: -L#{lib} -ltensorflow_c
+      Cflags: -I#{include}
+    EOF
+    mkdir_p(lib/"pkgconfig")
+    File.open(lib/"pkgconfig/tensorflow_c.pc", "w") { |f| f.write(pc) }
```

Comment:
Use `(lib/"pkgconfig/tensorflow_c.pc").write pc` instead. You don’t even need to make a variable for it.